foundations/javascript_basics/DOM-manipulation.md

```diff
@@ -368,18 +368,30 @@ You can find a more complete list with explanations of each event on [this page]
 
 ### Practice
 
-Manipulating web pages is the primary benefit of the JavaScript language!  These techniques are things that you are likely to be messing with _every day_ as a front-end developer, so let's practice!
+Manipulating web pages is the primary benefit of the JavaScript language! These techniques are things that you are likely to be messing with _every day_ as a front-end developer, so let's practice!
 
 1. Do the first exercise in Wes Bos's JavaScript30 program by cloning the repo at [https://github.com/wesbos/JavaScript30](https://github.com/wesbos/JavaScript30). Check out the [Video Tutorial](https://www.youtube.com/watch?v=VuN8qwZoego) for instructions on the project.
-2. Go back to your "Rock Paper Scissors" game from a previous lesson and give it a simple UI so that the player can play the game by clicking on buttons rather than typing their answer in a prompt.
-   1. Copy your original code into a new file so you don't lose it.
+2. Before we proceed to the next step, we need to get familiar with a very popular git workflow. [Read this article](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/) for everything that you need to know for now.
+   1. The above article shows us how to use a "fork and branch" workflow to contribute to another user's repo. But since we'll be adding a new feature to our own repo, we don't need to make a fork or add a remote since we already have it set up. We can start with the 'Making a branch' step.
```

Comment:
The formatting for the sub-bullets seems to start with `i, ii, iii`, etc, then switch to `1, 2, 3`, then back to `i, ii, iii` style notation. Let's standardize this. I think we use just numbers across the curriculum, even for numbered sub bullets.

Reply:
@twalton83 Not sure what you mean, I used numbers for sub bullets as well. Or are you saying I should use the alternating style?

Reply:
Look at the markdown preview, you’ll see what I’m talking about.

Reply:
Oh, I see it now. Shall I make it like the below snippet? I added a space before the period. Otherwise it seems to appear as numerals.

1. A
2. B
   1 . a
   2 . b
   3 . c
   4 . d